### PR TITLE
Added and enabled more Indian Languages

### PR DESCRIPTION
**Pull request by vinayaksharmagh**
I have added language codes for some additional Indian languages (Sanskrit, Bihari, Kashmiri and Pali) and have enabled them.

### CODE REVIEW

**Comment by Ostico** [comment]
Hi @vinayaksharmagh , thank you for your contribution, we will check if the codes match with our internal systems and in the case we will accept your PR.

**Comment by lorenzosecchi** [comment]
**Bihari**: Codes are the same as ISO standards
The language is absent on all our systems (Matecat, MyMemory and Core)

**Kashmiri**: Codes are the same as ISO standards
The language is absent on Matecat e Core
The language is present on MyMemory (codes are the same)

**Pali**: Codes are the same as ISO standards
The language is absent on all our systems (Matecat, MyMemory and Core)

**Sanskrit**: Codes are different from ISO standards
should be:
{
            "localized":[{
                "en":"Sanskrit"
            }],
            "isocode":"sa",
            "enabled":true,
            "rtl":false,
            "rfc3066code":"sa-IN"
        }
The language is absent on Matecat e Core
The language is present on MyMemory (codes are different, the correct ones are the ones in MyMemory as per ISO standards)

**Review by lorenzosecchi — COMMENTED** [review]
Bihari: Codes are the same as ISO standards
The language is absent on all our systems (Matecat, MyMemory and Core)

Kashmiri: Codes are the same as ISO standards
The language is absent on Matecat e Core
The language is present on MyMemory (codes are the same)

Pali: Codes are the same as ISO standards
The language is absent on all our systems (Matecat, MyMemory and Core)

Sanskrit: Codes are different from ISO standards
should be:
{
"localized":[{
"en":"Sanskrit"
}],
"isocode":"sa",
"enabled":true,
"rtl":false,
"rfc3066code":"sa-IN"
}
The language is absent on Matecat e Core
The language is present on MyMemory (codes are different, the correct ones are the ones in MyMemory as per ISO standards)

File: lib/Utils/Langs/supported_langs.json [hunk 1]
```diff
@@ -1834,6 +1864,16 @@
...
         }
+    ,
+	{
+            "localized":[{
+                "en":"Sanskrit"
+            }],
+            "isocode":"sn",
+            "enabled":true,
+            "rtl":false,
+            "rfc3066code":"sn-IN"
```
Comment (Ostico) [inline lib/Utils/Langs/supported_langs.json:1875]: Hi @vinayaksharmagh can you adjust this code to `sa-IN` and the isocode value to `sa`

**Review by Ostico — CHANGES_REQUESTED** [review]

File: lib/Utils/Langs/supported_langs.json (hunk 1 above)
Comment (vinayaksharmagh) [reply lib/Utils/Langs/supported_langs.json]: Yes, I have made the desired adjustments.
Please check.

https://github.com/matecat/MateCat/pull/1817/commits/f24037aa9df24dbdedef105552065f8558018fba

**Review by lorenzosecchi — APPROVED** [review]
Codes are allo OK now ;)

**Comment by vinayaksharmagh** [comment]
@Ostico If everything is fine, can you please accept the merge now?

File: lib/Utils/Langs/supported_langs.json (hunk 1 above)
Comment (Ostico) [reply lib/Utils/Langs/supported_langs.json]: Perfect thank You.

**Review by Ostico — APPROVED** [review]

**Comment by Ostico** [comment]
Hi @vinayaksharmagh we will accept for sure your PR, but we have to adjust our internal systems accordingly before the merge.

**Pull request merged**